packages/commcare/test/index.js

```diff
@@ -97,3 +96,74 @@ describe('SubmitXls', () => {
     expect(formdata.get('create_new_cases')).to.equal('on');
   });
 });
+
+describe('getCases', () => {
+  it('should fetch cases', async () => {
+    testServer
+      .intercept({
+        path: `/a/${domain}/api/v0.5/case`,
+        method: 'GET',
+      })
+      .reply(200, () => {
+        // simulate a return from commcare
+        return {
+          data: [
+            {
+              case_id: '12345',
+              properties: { case_type: 'pregnancy', case_name: 'Jane' },
+            },
+          ],
+        };
+      });
+
+    const state = {
+      configuration: {
+        hostUrl,
+        applicationName: domain,
+        appId: app,
+        username: 'user',
+        password: 'password',
+      },
+    };
+
+    const { data } = await execute(get('case'))(state);
+
+    // The response  should be on state.data
+
+    expect(data.data[0]).to.haveOwnProperty('case_id');
```

Comment:
Since we know the value of `case_id`, we may as well assert the actual value:
```
expect(data.data[0].case_id).to.equal('12345')
```

And actually before doing that I would assert on the length of state.data.
````
expect(data.length).to.equal(1)
```